packages/fluentui/docs/src/examples/components/List/Variations/ListExampleTruncate.shorthand.tsx

```diff
@@ -11,12 +11,14 @@ const actions = (
         iconOnly: true,
         text: true,
         title: 'Check',
+        key: 'check',
```

Comment:
as with other cases, maybe we should try to make these unique, maybe include the name of the example in their value, like 'example-truncate-check` or something.

Reply:
It's unique in this list.